src/components/AddressAvatar.tsx

```diff
@@ -1,23 +1,19 @@
 import useRandomColor from '@/hooks/useRandomColor'
 import { cx } from '@/utils/class-names'
-import * as bottts from '@dicebear/bottts'
-import { createAvatar } from '@dicebear/core'
 import Image from 'next/image'
-import { ComponentProps, forwardRef, useMemo } from 'react'
+import { ComponentProps, forwardRef } from 'react'
 
 export type AddressAvatarProps = ComponentProps<'div'> & {
   address: string
+  avatar: string
 }
 
 const AddressAvatar = forwardRef<HTMLDivElement, AddressAvatarProps>(
-  function AddressAvatar({ address, ...props }: AddressAvatarProps, ref) {
+  function AddressAvatar(
+    { address, avatar, ...props }: AddressAvatarProps,
+    ref
+  ) {
     const backgroundColor = useRandomColor(address, 'dark')
-    const avatar = useMemo(() => {
```

Comment:
I think you can make this component take avatar as optional, and if the avatar is not provided, use the avatar bottts generator instead of generating the avatar directly inside the `my-account` store.

That way, you can differentiate the one that uses bottts avatar and not, so you can make the p-[7.5%] that you deleted be only applied to bottts avatar

Anyway, there is an issue with the current implementation, where the avatar all becomes the same.

Reply:
Yes, i believe it's because I made the avatarAddress accept an avatar instead of letting it generate based on the address passed. So all avatar are generating address belonging to the logged in user. I've resolved it and also added the `p-[7.5%]` back

Reply:
for this, you forgot to use the userAvatar anywhere,
should put it in the Image src in AddressAvatar
```
src={userAvatar ?? avatar}
```

and will be better if you remove the renaming of avatar in props, instead, rename the avatar that uses bottts to defaultAvatar

so
```
src={avatar ?? defaultAvatar}
```
I think it more understandable

Reply:
Definitely, I must have missed passing the userAvatar into the component. 

I think it's a good idea to just name it the `generatedAvatar`. It'll be understandable in that regard